cypress/e2e/homeworks/hw_page_objects/eshop_registration.cy.js

```diff
@@ -0,0 +1,26 @@
+import { HomePage } from "../../../page-objects/eshop/home_page";
+import { Registration } from "../../../page-objects/eshop/registration_page";
+
+describe("Open eshop and Fill in Registration Form", () => {
+  it("Open eshop and click My Account", () => {
+    const homePage = new HomePage();
+    homePage.openEshop();
+    homePage.clickMyAccountButton();
+    homePage.clickRegisterButton();
+  });
+
+  it.only("Fill in Registration Details", () => {
```

Comment:
Only by se ti nikdy nemělo dostat do GitHubu. Jeho použití by mělo být omezené jen na lokální vývoj, protože může spoustu věcí rozbít při vzdáleném spouštění.